check_env.py

```diff
@@ -12,12 +12,12 @@
 # Set both min and max versions to avoid ambiguity.
 # This should match environment.yml file.
 PKGS = {'jupyter': None,
-        'numpy': ('1.24', None),
-        'matplotlib': ('3.2', None),
-        'jupyterlab': ('3.1', None),
-        'astropy': ('5.2', None),
-        'scipy': ('1.0', None),
-        'pyvo': ('1.4', None),
+        'numpy': ('1.26', None),
+        'matplotlib': ('3.8', None),
+        'jupyterlab': ('4.0', None),
+        'astropy': ('6.0', None),
```

Comment:
Not sure whether we can do this yet, I think we need to disentangle this version check from the conda environment versions. 

E.g. the notebooks should work with older astropy, no need to get the latest version (especially that a lot of packages in the ecosystem is still incompatible with astropy6).

So I would keep this list for the minimum versions, we do check for these in CI anyway, and the conda env separate.

Reply:
@bsipocz so are you recommending to set the environment check to, say, version 5.3, and leave the `environment.yml` entry for astropy as-is?

Reply:
or even leave 5.2 and we bump versions when we run into limitations (or when they are getting a bit out of date).

Totally separately you can update the versions in `environment.yml` as that is a combination users are expected to setup for the workshop itself (in my mind, the notebooks should be able to run separately from that specific environment, and indeed we test with the minimum versions separatetly)

Reply:
I'm sorry I wasn't clear. This comment is true for all the minimum versions, bump them when needed or when SPEC0 or similar would say they are outdated, but don't require latest and greatest by default.